BCStudentSoftwareDevTeam/celts
# Countdown on event view

Issue: We are trying to add a relative date to the Event View, just like a countdown before the event starts. (fixes issue #998)

Did: We calculated and displayed a countdown to the specified event, considering both the date and time of the event. Depending on the remaining time, it shows different messages indicating how soon the event is happening.

Test:

- Switch to branch "countdownOnEventView"
- reset the database to test
- Select a term on the drop-down of the event list (I usually just pick fall 2021) or you can just create an event
-  View the event and check if the countdown is accurate 
- Review the code
